scripts/schema-publish.sh

```diff
@@ -32,12 +32,9 @@ publish_schema() {
   local target=$deploydir/$base/$date
 
   mkdir -p $deploydir/$base
-  # replace the WORK-IN-PROGRESS placeholders
-  sed -e $sedCmd $schemaDir/$schema > $target.yaml
 
-  node scripts/yaml2json/yaml2json.js "$target.yaml"
-  rm "$target.yaml"
-  mv "$target.json" "$target"
+  # replace the WORK-IN-PROGRESS placeholders
+  sed ${sedCmd[@]} $schemaDir/$schema | npx yaml --json --indent 2 --single > $target
```

Comment:
The `--json` flag transforms the streamed yaml to JSON.
The `--indent 2` flag specifies the [indentation level of the JSON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#space)
The `--single` flag indicates we are processing a single file and not a stream, which prevents the output from being wrapped in an extra `[]` pair.